unit_tests/falco/test_configuration.cpp

```diff
@@ -133,52 +136,81 @@ TEST(Configuration, configuration_environment_variables)
         "    sample_list:\n"
         "        - ${ENV_VAR}\n"
         "        - ' ${ENV_VAR}'\n"
-        "        - $UNSED_XX_X_X_VAR\n";
-    conf.load_from_string(sample_yaml);
+	"        - '${ENV_VAR} '\n"
```

Comment:
Tests were expanded:
* new case with whitespace AFTER the expanded string. Moreover, since we now support env variables embedded inside bigger strings, both ` ${ENV_VAR}` and `${ENV_VAR} ` are now expanded
* we do now tests that env variables found in the middle of strings are properly expanded (`paths[0,1,2,3]` cases)
* `paths[4]` checks an even weirder case, ie: when the env variable gets expanded to another env variable. It must get resolved twice.

Reply:
We check that `plugins` config (that is getted as a sequence)  is still correctly expanded.